docs/modules/servers/pages/distributed/operate/metrics_es.adoc

```diff
@@ -0,0 +1,195 @@
+= Distributed James Server &mdash; Metrics

```

Comment:
Why this file speaks about dropwizard in a file about ElasticSearch metric reporting?

Reply:
It was there at the first place lol. I'm cleaning the document.